smart_room/config.py

```diff
@@ -0,0 +1,54 @@
+import os
```

Comment:
You can add a blank line after built-in package

Reply:
Fix in 1d4e9ef.